package.json

```diff
@@ -21,19 +16,21 @@
     "lint-python": "flake8 tools loadtest",
     "preinstall": "node bin/pre-install.js",
     "build:clean": "rimraf dist && mkdirp dist",
-    "build": "babel src --out-dir dist",
+    "build": "babel --extensions \".ts\" src --out-dir dist",
     "generate-version-file": "node bin/generate-version-file.js",
     "docker:build": "docker build -t $npm_package_name:dev .",
     "docker:run": "node bin/docker-run --name $npm_package_name $npm_package_name:dev",
     "docker:run:detached": "yarn docker:run --detach",
     "docker:stop": "docker container stop $npm_package_name",
     "serve": "node dist/index.js",
-    "start": "nodemon --exec babel-node -w .env -w src src/index.js",
+    "start": "nodemon --exec babel-node --extensions \".ts\" -w .env -w src src/index.ts",
     "start:prod": "run-s build:clean build serve",
     "test": "jest",
     "test-python": "./tools/test_decode_jwt_payload.py",
     "test-lockfile": "lockfile-lint --path yarn.lock --allowed-hosts yarn --validate-https",
-    "test-all": "run-p flow lint test test-python test-lockfile license-check"
+    "test-all": "run-p ts lint test test-python test-lockfile license-check",
+    "ts": "tsc",
```

Comment:
small nit, I wonder about naming the script `tsc`, but I'm not sure what the convention in other projects is.

Reply:
let's keep it this way, we can always change it later if we're bored :-)